xbmc/cores/VideoPlayer/VideoRenderers/LinuxRendererGL.cpp

```diff
@@ -452,17 +454,24 @@ void CLinuxRendererGL::LoadPlane(YUVPLANE& plane, int type,
     glBindBufferARB(GL_PIXEL_UNPACK_BUFFER_ARB, 0);
 }
 
-void CLinuxRendererGL::Flush()
+bool CLinuxRendererGL::Flush(bool saveBuffers)
 {
+  bool safe = saveBuffers && CanSaveBuffers();
```

Comment:
save, not safe;-)

Reply:
here it is "safe" because I mean the state (!= verb)

Reply:
Ooops. Okay, then.